vehicle/src/main/scala/com/wedriveu/vehicle/control/VehicleBehavioursImpl.scala

```diff
@@ -102,6 +110,20 @@ class VehicleBehavioursImpl(selfDrivingVehicle: SelfDrivingVehicle) extends Vehi
       drainBattery()
       timePassedForBattery += timeStepBatteryConsumed
     }
+    stopUi.writeMessageLog("New Position = " + latInter + " , " + lonInter)
```

Comment:
Also here, put message strings in constants and if you like use string formatting like "New Position = %s , %s" or something like that (because i don't know if this is correct).